device-types/Cisco/C1000-24P-4G-L.yaml

```diff
@@ -3,66 +3,115 @@ manufacturer: Cisco
 model: C1000-24P-4G-L
 slug: c1000-24p-4g-l
 part_number: C1000-24P-4G-L
+comments: '[Cisco Catalyst 1000 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-1000-series-switches/nb-06-cat1k-ser-switch-ds-cte-en.html)'
 u_height: 1
 is_full_depth: false
 console-ports:
-  - name: Serial Console
+  - name: con0
     type: rj-45
-  - name: USB Console
-    type: usb-mini-b
+  - name: USB con0
+    type: usb-a
 power-ports:
   - name: Built-in
```

Comment:
Current documentation addresses the PSUs as PSU1, PSU2, etc. Please revert this to PS-1 or PSU1 (my preference is PSU1), or provide documentation where it is called Build-in. Thanks!